README.md

```diff
@@ -31,39 +31,41 @@ This library was constructed using Kotlin, and as such used several idiomatic Ko
 
 *Note*: These examples can be composed with each other in arbitrary ways to render the structured text.
 
-### Multi-line Uniform Text
+### `CellFormatter`: Multi-line Uniform Text
 
 #### Kotlin API
 
-Let the pre-constructed line values be:
+<details>
+    <summary>Let the pre-constructed line values be:</summary>
+
 ```kotlin
 val line1 = faker.getWords(5)
 val line2 = faker.getWords(2)
 val line3 = faker.getWords(3)
 ```
 
-Then, these values can be used in the following ways:
+</details>
+
+<details>
```

Comment:
that's cool, didn't know they supported expanded elements like that.

Reply:
They are cool. though my IntelliJ plugin doesn't recognize them, which is unfortunate. Also, the headers used for them are somewhat limited. I.e. you can't use the markdown formatting on the summary lines.